test/functional/cmdLineTests/criu/playlist.xml

```diff
@@ -87,6 +87,13 @@
 	</test>
 	<test>
 		<testCaseName>cmdLineTester_criu_nonPortableRestore</testCaseName>
+		<disables>
+			<disable>
+				<comment>Metronome is not supported on z</comment>
+				<variation>-Xgcpolicy:metronome</variation>
+				<platform>s390x_linux</platform>
```

Comment:
Given that this is only supported on AIX and Linux on x86, @ymanton  do we need to also disable this for PPCLE?

Reply:
Yes I Believe so.

Reply:
@VermaSh  Can you make changes to disable this for PPCLE as well?